### PR TITLE
Current version

### PR DESCRIPTION
**Pull request by sijad**
Sorry if seems ugly

### CODE REVIEW

**Comment by fffw** [comment]
Thanks @sijad, it looks elegant!

A few issues:

1. Typo. `vesrion` should be `version`.

2. It doesn't show version correctly on my Chrome. Maybe use a `<span>` to hold the version number separately instead of `%s` formatter? Copy the element here:

    `<p id="vesrion-number" data-localize="CURRENT_VERSION" data-version="2.0.10" class="centered vesrion-number">Currently v%s</p>`

    `<p id="vesrion-number" data-localize="CURRENT_VERSION" data-version="2.0.10" class="centered vesrion-number">نسخه فعلی %s</p>`

3. It would be good to hide the whole element until we successfully retrieved version number from GitHub.

**Comment by sijad** [comment]
I've fixed 1 and 3, but not sure about second one, you mean this:
`<p id="vesrion-number" data-localize="CURRENT_VERSION" data-version="2.0.10" class="centered vesrion-number">Currently v<span>%s</span></p>`
or move version out like this:
`<p id="vesrion-number" data-localize="CURRENT_VERSION" data-version="2.0.10" class="centered vesrion-number">Currently v</p><span class="current-version-number">%s</span>`

**Comment by fffw** [comment]
Oh I mean to make the translatable text as an element, and the version number from AJAX as another one, so we can handle them separately.

Through this way, we don't need to care about translation after retrieved version number, hence make js code simpler. Something like below (text can vary):

```
<p class="centered current-version">
    <span data-localize="CURRENT_VERSION">Current version: </span>
    <span id="version-number"></span>
</p>
```

Does it sound natural to you @sijad ?

File: getlantern.org/static/css/main.css [hunk 1]
```diff
@@ -456,6 +463,10 @@ section#faq {
   a.download-desktop {
     display: none;
   }
+  
+  p.version-number {
+	  display: none !important;
```
Comment (fffw) [inline getlantern.org/static/css/main.css]: duplicate with L149?

Comment (sijad) [inline getlantern.org/static/css/main.css:468]: I've used jQuery to hide/show `.version-number`, for hiding it in mobile view we need `!important` to make sure browser ignore inline style `display: block;` which jQuery added to element

Comment (fffw) [inline getlantern.org/static/css/main.css]: Oh sorry I didn't notice it's for mobile view. It works great!

**Comment by sijad** [comment]
@fffw as you suggested using `span`, it's way cleaner!

**Comment by fffw** [comment]
LGTM. Cool thanks @sijad !

**Pull request merged**